cypress/integration/vizsurvey.spec.js

```diff
@@ -1,26 +1,26 @@
 /// <reference types="cypress" />
 
-describe('vizsurvey', () => {
```

Comment:
Just curious, is this automatically changing from vscode? Just would like to make my future commits similar so that you don't have these auto fixes in your future PRs.

Reply:
Those changes got generated automatically.  I don't recall editing the file.  I do have eslint setup and it may have auto formatted it?